README.rst

```diff
@@ -0,0 +1,40 @@
+PyLadies Sphinx Theme
+=====================
+
+A PyLadies theme for docs that use Sphinx_/`Read the Docs`_.
+
+.. _Sphinx: http://sphinx-doc.org/
+.. _Read the Docs: https://readthedocs.org/
+
+
+How to use this theme
+---------------------
+
+To use this theme:
+
+#. Install the theme.
+   Run ``pip install pyladies_sphinx_theme``.
```

Comment:
I think with pip, it is (or should be) `pyladies-sphinx-theme` then gets translated into underscores for import/use.